SwiftCompilerSources/Sources/SIL/FunctionConvention.swift

```diff
@@ -25,7 +25,7 @@ public struct FunctionConvention : CustomStringConvertible {
   let bridgedFunctionType: BridgedASTType
   let hasLoweredAddresses: Bool
 
-  init(for bridgedFunctionType: BridgedASTType, in function: Function) {
+  public init(for bridgedFunctionType: BridgedASTType, in function: Function) {
```

Comment:
Why does this need to be public?

Reply:
I was doing something not quite right. It doesn't anymore.